files/test_task_helper.py

```diff
@@ -95,3 +95,27 @@ def task(self, args):
         MyTask().run()
         result = json.loads(sys.stdout.getvalue())
         self.assertEqual(result, {'result': 'world'})
+
+    def test_debug_method(self):
+        stub_stdin(self, '{"hello": "world"}')
+        stub_stdouts(self)
+        class MyTask(TaskHelper):
+            def task(self, args):
+                self.debug('a debugging statement')
+                self.debug('another debugging statement')
+                raise TaskError(
+                    'a task error',
+                    'mytask/failed',
+                    { 'debug': ['a debugging statement', 'another debugging statement'] }
```

Comment:
Should this be `debug: self.debug_statements`? Instead of hard-coding the list?

Reply:
Ah, yes. Thanks!